webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -12,6 +12,7 @@ export type Props = {
     className?: string;
     onExited: () => void;
     modalHeaderText?: React.ReactNode;
+    modalSubheadingText?: React.ReactNode;
```

Comment:
Nit, header vs heading are different things, so this should probably refer to it as the `subheader`. The header is the section at the top of a page while a heading is like a title in a document.

I only learned the difference a few months ago, but I've resisted the urge of looking through all the code and finding where to change it 😅

Reply:
That said, what would be your thoughts on having separate props for `modalHeaderText` and `modalSubheaderText` vs just having the `DateTimeInput` pass both "Schedule message" and "(UTC+05:45) Kathmandu" with the wrapping div in as the `modalHeaderText`? I'm trying to think of where else we'd want a modal to have two lines in the header since we haven't needed that elsewhere yet

Reply:
Done.

Reply:
I'd prefer separate props as the CSS for the wrapper div is dependent on the styling of modal's header. If one changes without other, both will break. Keeping them tied together is important here.